backend/main/strategies/token_weighted_default.go

```diff
@@ -0,0 +1,63 @@
+package strategies
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/brudfyi/flow-voting-tool/main/models"
+)
+
+type TokenWeightedDefault struct{}
+
+func (s *TokenWeightedDefault) TallyVotes(votes []*models.VoteWithBalance, proposalId int) (models.ProposalResults, error) {
+	var r models.ProposalResults
+	r.Results = map[string]int{}
+	r.Results_float = map[string]float64{}
+	r.Proposal_id = proposalId
+
+	//tally votes
+	for _, vote := range votes {
+		r.Results[vote.Choice] += int(float64(*vote.PrimaryAccountBalance) * math.Pow(10, -8))
+		r.Results_float[vote.Choice] += float64(*vote.PrimaryAccountBalance) * math.Pow(10, -8)
+	}
+
+	return r, nil
+}
+
+func (s *TokenWeightedDefault) GetVotes(votes []*models.VoteWithBalance, proposal *models.Proposal) ([]*models.VoteWithBalance, error) {
+
+	for _, vote := range votes {
+		weight, err := s.GetVoteWeightForBalance(vote, proposal)
+		if err != nil {
+			return nil, err
+		}
+		vote.Weight = &weight
+	}
+
+	return votes, nil
+}
+
+func (s *TokenWeightedDefault) GetVoteWeightForBalance(vote *models.VoteWithBalance, proposal *models.Proposal) (float64, error) {
+	var weight float64
+	var ERROR error = fmt.Errorf("no weight found, address: %s, strategy: %s", vote.Addr, *proposal.Strategy)
+
+	if vote.PrimaryAccountBalance == nil {
+		return 0.00, nil
+	}
+
+	weight = float64(*vote.PrimaryAccountBalance) * math.Pow(10, -8)
+
+	switch {
+	case proposal.Max_weight != nil && weight > *proposal.Max_weight:
+		weight = *proposal.Max_weight
+		return weight, nil
+	case proposal.Max_weight != nil && weight < *proposal.Max_weight:
```

Comment:
since we are using Max_weight now to implement an option cap, can you update your latest migration (or add a new one) to remove the `token-weighted-capped` strategy from the `voting_strategies` table?